pkgs/tools/misc/bonsai/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv
+, lib
+, fetchFromSourcehut
+, gitUpdater
+, hare
+, hare-ev
+, hare-json
+}:
+
+stdenv.mkDerivation rec {
+  pname = "bonsai";
+  version = "1.0.0";
```

Comment:
1.0.2 was released by now.

Reply:
hare in nixpkgs isn't new enough for 1.0.2

Reply:
1.0.0 is the latest version of bonsai which is compatible with the `harec` currently in nixpkgs. i can attempt a harec update in the future (i've updated the nixpkgs harec in the past), but i'm hoping that can be separate from this PR

Reply:
In that case it's fine.